static/backbone/js/test/spec/data-manager-test.js

```diff
@@ -0,0 +1,199 @@
+define(["underscore",
+        "collections/markers",
+        "models/marker",
+        "../../test/spec-helper"],
+    function (_, Markers, Marker) {
+        'use strict';
+
+        var getCount = function (collections) {
+                var count = 0;
+                _.map(collections, function (collection) {
+                    collection.each(function (model) {
+                        if (model.get("isVisible")) {
+                            ++count;
+                        }
+                    });
+                });
+                return count;
+            };
+
+        /**
+         * Note, these tests do not test:
+         *  - updateCollection ('cause it's private)
+         *  - fetchProjects ('cause it queries the api)
+         *  - fetchDataByProjectID ('cause it queries the api)
+         *  TODO: there's a way to fake a server w/Backbone + Jasmine. Look into it.
```

Comment:
just need to make some mocks - http://sinonjs.org/ is one option but I'm sure there are many